modules/client/src/internal/utils.ts

```diff
@@ -237,6 +245,48 @@ export function toTokenTransfer(transfer: SubgraphTransferListItem): Transfer {
   }
 }
 
+export function toPluginRepoRelease(
+  release: SubgraphPluginRepoReleaseListItem,
+  metadata: PluginRepoReleaseMetadata,
+): PluginRepoRelease {
+  return {
+    release: release.release,
+    currentBuild: Math.max(...release.builds.map((build) => build.build)),
+    metadata,
+  };
+}
+
+export function toPluginRepoListItem(
+  pluginRepo: SubgraphPluginRepoListItem,
+  releases: PluginRepoRelease[],
+): PluginRepoListItem {
+  return {
+    address: pluginRepo.id,
+    subdomain: pluginRepo.subdomain,
+    releases,
+  };
+}
+export function toPluginRepo(
+  pluginRepo: SubgraphPluginRepo,
+  releaseMetadata: PluginRepoReleaseMetadata,
+  buildMetadata: PluginRepoBuildMetadata,
+): PluginRepo {
+  return {
+    address: pluginRepo.id,
+    subdomain: pluginRepo.subdomain,
+    current: {
+      build: {
+        metadata: buildMetadata,
+        number: pluginRepo.releases[0].builds[0].build,
+      },
+      release: {
+        metadata: releaseMetadata,
+        number: pluginRepo.releases[0].release,
```

Comment:
It would be nice to use the conditional syntax, just in case:

`obj?.prop?.[0]?.value`

Reply:
I can add it but this wont be undefined since one plugin hast at least one release and a release has at least one build

Reply:
Better be safe than sorry, a new subgraph version might potentially alter something